pages/common/xprop.md

```diff
@@ -0,0 +1,20 @@
+# xprop
+
+> A tool for displaying window and font properties in an X server.
+> More information: <https://manned.org/xprop>.
+
+- Display the name of the root window:
+
+`xprop -root {{WM_NAME}}`
+
+- Display the window manager hints for a window:
+
+`xprop -name {{window_name}} {{WM_HINTS}}`
```

Comment:
Could you give any syntax for WM_HINTS?

Reply:
It's the name of the X property, usually something like `WM_FOO`.